lib/Xmds/Soap.php

```diff
@@ -1994,7 +1976,7 @@ protected function checkBandwidth($displayId)
         // Uncomment to enable auditing.
         //$this->logProcessor->setDisplay(0, true);
 
-        $this->display = $this->displayFactory->getById($displayId);
+        $this->display = $this->displayFactory->getById($displayId, true);
```

Comment:
So I can see why you did this and I understand that it makes the tag lookup more elegant for submitting stats - however `checkBandwidth` is called for every XMDS method, so adding this here means we have made **all** the XMDS calls more complicated to save a lookup in the stats call.

This probably isn't a good trade-off in this case.